playwright/support/fixtures/editor-api.ts

```diff
@@ -45,17 +47,32 @@ export const test = base.extend<EditorApiFixture>({
 					container.style.background = 'white'
 					document.body.appendChild(container)
 
+					const method = type === 'editor' ? 'createEditor' : 'createTable'
 					// @ts-expect-error - OCA.Text is a global
-					await window.OCA.Text.createEditor({
+					await window.OCA.Text[method]({
 						el: container,
 						...(fileId != null ? { fileId } : { content }),
 						readOnly,
 					})
 				},
-				{ containerId, content: options.content, fileId: options.fileId, readOnly: options.readOnly },
+				{ containerId, type: options.type, content: options.content, fileId: options.fileId, readOnly: options.readOnly },
 			)
 		}
 
+		const createEditor = async (
+			options: {
+				type: 'editor' | 'table',
+				content?: string,
+				fileId?: number,
+				readOnly?: boolean,
+			},
+		) => {
+			await createComponent(options)
+		}
+
```

Comment:
This seems like `createEditor` does the same as `createComponent` or am I missing something?